nicolalandro_ntsnet-cub200_ntsnet.md

```diff
@@ -0,0 +1,120 @@
+---
+layout: hub_detail
+background-class: hub-background
+body-class: hub
+category: researchers
+title: ntsnet
+summary: a fine grane model for image classification.
```

Comment:
"a fine grane model for image classification." -> "a fine grained model for bird image classification"

Reply:
Scuse me again, I will correct it!